File: js/org/jmol/awtjs/AjaxURLStreamHandler.js
Clazz.declarePackage ("org.jmol.awtjs");
Clazz.load (["java.net.URLStreamHandler"], "org.jmol.awtjs.AjaxURLStreamHandler", null, function () {
c$ = Clazz.decorateAsClass (function () {
this.protocol = null;
Clazz.instantialize (this, arguments);
}, org.jmol.awtjs, "AjaxURLStreamHandler", java.net.URLStreamHandler);
Clazz.makeConstructor (c$, 
function (protocol) {
Clazz.superConstructor (this, org.jmol.awtjs.AjaxURLStreamHandler, []);
this.protocol = protocol;
}, "~S");
Clazz.defineMethod (c$, "openConnection", 
function (url) {
return null;
}, "java.net.URL");
});

File: src/org/jmol/awtjs/AjaxURLStreamHandler.java
package org.jmol.awtjs;

import java.io.IOException;
import java.net.URL;
import java.net.URLConnection;
import java.net.URLStreamHandler;

/**
 * 
 * A method to allow a JavaScript Ajax 
 * 
 */
public class AjaxURLStreamHandler extends URLStreamHandler {

	String protocol;

	public AjaxURLStreamHandler(String protocol) {
		this.protocol = protocol;
	}

	@Override
	protected URLConnection openConnection(URL url) throws IOException {
		return null;
	}


}
